etc/org.opencastproject.assetmanager.aws.s3.AwsS3AssetStore.cfg

```diff
@@ -26,5 +26,5 @@ org.opencastproject.assetmanager.aws.s3.enabled=false
 # Enable path-style for URL adressing
 # Leave this commented out to use the default style for AWS S3
 # If you are using another S3 service/provider, please refer to its documentation.
-#org.opencastproject.assetmanager.aws.s3.path.style=true
+#org.opencastproject.assetmanager.aws.s3.path.style=false
```

Comment:
Maybe explicitly state what the default value is. We so that at many other places by now:

```suggestion
# Default: false
#org.opencastproject.assetmanager.aws.s3.path.style=false
```